modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -1150,4 +1150,37 @@ definition {
 		ObjectAdmin.assertRequiredErrorPresent();
 	}
 
+	@description = "LPS-171802- Verify if the Condition card is hidden when are using the trigger On Subscription Status Update"
```

Comment:
Typo: Remove the "are" so it's 
LPS-171802- Verify if the Condition card is hidden when using the trigger On Subscription Status Update.